### DISCUSSION TITLE
Frontend autoupdate / refresh of values like LQI...

### DISCUSSION THREAD

**Discussion by crapp** [Koenkk/zigbee2mqtt#9813] [General]
Hi,

I am using the latest Version of zigbee2mqtt. I have deployed this on a server using docker and it works great. I was using an older cc2531 stick at first but switched to a cc2652RB from slae.sh which works much better. So far I am still experimenting and playing with the system and zigbee in general. One thing I have noticed since I switched to the cc2652RB stick the values on the frontend don't autoupdate anymore. This could be coincidental and I have done something completely different.

What I mean? Well values like LQI or the actions on my philips remote 

![image](https://user-images.githubusercontent.com/2805076/143037453-80fa6287-73ea-4aa9-861b-2122215009fc.png)

When I press a button on that remote I can see it works because it is logged to the logfile but the action on the frontend is always N/A and this did work before. What do I have to do to get the old behavior / autorefresh back?

Regards

**Comment by sjorge** [comment]
If you have homeassistent enabled it clears the action and a few other values, otherwise HA won't work. I'm guessing this is the case here.

**Reply by crapp** [reply]
@sjorge Thanks for your suggestion but I am not using homeassistant. I am using openhab. homeassistant is disabled in the settings
